mne_qt_browser/_pg_figure.py

```diff
@@ -2213,18 +2213,18 @@ def __init__(self, **kwargs):
                                             'Display each channel with its '
                                             'channel-type color.</li>'
                                             '<li>zscore:<br>'
-                                            ' Display the zscore for the '
+                                            'Display the zscore for the '
                                             'data from each channel across '
                                             'time. '
-                                            'Red indicates high z - scores,'
-                                            'Blue indicates low z - scores '
-                                            ' whilethe boundaries of the '
-                                            ' color gradientare defined by '
-                                            ' the minimum/maximum z-score. '
+                                            'Red indicates high z - scores, '
+                                            'blue indicates low z - scores, '
+                                            'and the boundaries of the '
+                                            'color gradient are defined by '
+                                            'the minimum/maximum z-score. '
                                             'This only works if precompute '
-                                            'is set to "True" or it is '
+                                            'is set to "True", or if it is '
                                             'enabled with "auto" and enough '
-                                            'free RAM.</li>'
+                                            'free RAM is available.</li>'
```

Comment:
seems like a good candidate for extracting the tooltip into a variable, so that it doesn't need to be so deeply indented and take up 18 lines

Reply:
Is https://github.com/mne-tools/mne-qt-browser/pull/28/commits/78e3964eecc5a097e23471dc9bbbac5b5c3bd2cf what you had in mind approximately? I didn't optimize this for space because I thought this is a good compromise between human readable (with the html tags) and less space than before (and obviously no more typos/formatting issues).

Reply:
btw, the "real" changes are now no longer highlighted in the diff. Check out https://github.com/mne-tools/mne-qt-browser/pull/28/commits/d190b0b28717b0dbb6d827ee65a1ef60a1183958 to see those.

Reply:
yep, that's what I meant.  The line savings / clarity is not as high as I'd hoped, but it still seems like an improvement to me.